alerts/monquorum.libsonnet

```diff
@@ -7,8 +7,8 @@
           {
             alert: 'CephMonQuorumAtRisk',
             expr: |||
-              count(ceph_mon_quorum_status == 1) <= ((count(ceph_mon_metadata) %s 2) + 1)
-            ||| % '%',
+              count(ceph_mon_quorum_status{%s} == 1) <= ((count(ceph_mon_metadata{%s}) %s 2) + 1)
+            ||| % [$._config.cephExporterSelector, $._config.cephExporterSelector, '%'],
```

Comment:
Is this change made assuming that there will be multiple exporter jobs working at the same time  ?

Reply:
Yes, I think it makes sense, as if you have 2 clusters monitored with same Prometheus the current version wouldn't work